docs/src/snforge-advanced-features/debugging.md

```diff
@@ -96,3 +103,80 @@ stack backtrace:
+│  ├─ [selector] execute_calls
+│  │  ├─ [contract name] SimpleContract
+│  │  ├─ [calldata] array![RecursiveCall {{ contract_address: ContractAddress(0x28f58bf524dc0adcf7468c67d7ffdac1e5d885d347c6a498978f538984dbda), payload: array![] }}, RecursiveCall {{ contract_address: ContractAddress(0x28f58bf524dc0adcf7468c67d7ffdac1e5d885d347c6a498978f538984dbda), payload: array![] }}]
+│  │  ├─ [call result] success: array![RecursiveCall {{ contract_address: ContractAddress(0x28f58bf524dc0adcf7468c67d7ffdac1e5d885d347c6a498978f538984dbda), payload: array![] }}, RecursiveCall {{ contract_address: ContractAddress(0x28f58bf524dc0adcf7468c67d7ffdac1e5d885d347c6a498978f538984dbda), payload: array![] }}]
+│  │  ├─ [selector] execute_calls
+│  │  │  ├─ [contract name] SimpleContract
+│  │  │  ├─ [calldata] array![]
+│  │  │  └─ [call result] success: array![]
+│  │  └─ [selector] execute_calls
+│  │     ├─ [contract name] SimpleContract
+│  │     ├─ [calldata] array![]
+│  │     └─ [call result] success: array![]
+│  └─ [selector] execute_calls
+│     ├─ [contract name] SimpleContract
+│     ├─ [calldata] array![]
+│     └─ [call result] success: array![]
+└─ [selector] fail
+   ├─ [contract name] SimpleContract
+   ├─ [calldata] array![0x1, 0x2, 0x3, 0x4, 0x5]
+   └─ [call result] panic: (0x1, 0x2, 0x3, 0x4, 0x5)
+```
+</details>
+<br>
+
+---
+
+## Trace Output Explained
+
+Here's what each tag in the trace represents:
+
+| Tag                  | Description                                                                                                                                               |
+|----------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `[test name]`        | The path to the test being executed, using the Cairo module structure. Indicates which test case produced this trace.                                     |
+| `[selector]`         | The name of the contract function being called. The structure shows nested calls when one function triggers another.                                      |
+| `[contract name]`    | The name of the contract where the selector (function) was invoked. Helps trace calls across contracts.                                                   |
+| `[entry point type]` | (Only in detailed view) Type of entry point used: External, Constructor, L1Handler. Useful to differentiate the context in which the call is executed.    |
+| `[calldata]`         | (Only in standard view) The arguments passed into the function call.                                                                                      |
```

Comment:
nit: I know you mean it's possible in view > standard (standard and full), but could we rephrase it?